docker/config/thriftserver-hive-config_local.json

```diff
@@ -0,0 +1,8 @@
+{
+  "databaseService" : "hive2",
+  "databaseHostName" : "spark-thriftserver",
+  "databasePort" : "10000",
```

Comment:
Should this be 10001 as thats the port we are exposing in docker-compose from thrift-server container for database.

Reply:
No - because it's container-to-container communication within the same network, you use the container port and not the host-visible port.